github/pulls_test.go

```diff
@@ -349,7 +349,7 @@ func TestPullRequestsService_Merge(t *testing.T) {
 			}`)
 	})
 
-	merge, _, err := client.PullRequests.Merge("o", "r", 1, "merging pull request")
+	merge, _, err := client.PullRequests.Merge("o", "r", 1, "merging pull request", false)
```

Comment:
You probably want to add:
`testHeader(t, r, "Accept", mediaTypeSquashPreview)`
right after the `testMethod` above like is done in `repos_test.go` line 22